🚑️ Update DMC to MC_X motors

FluffyFTW opened the pull request:
- Update the DMC motors to MC_X motors
- Changed i2c line to 1 for the pca9685
- Updated ESP to connect to AWS server

FluffyFTW committed:
🚑️ Update DMC to MC_X motors

- Update the DMC motors to MC_X motors
- Changed i2c line to 1 for the pca9685
- Updated ESP to connect to AWS server

FluffyFTW committed:
🚑️ Update DMC to MC_X motors

- Update the DMC motors to MC_X motors
- Changed i2c line to 1 for the pca9685
- Updated ESP to connect to AWS server

FluffyFTW committed:
cleaned up warnings

FluffyFTW committed:
i2c channel is now set to ch2

FluffyFTW committed:
Merge branch 'main' into rmd-to-mcx

Coreyboy1820 commented on code/applications/control_loops/drive_serial.cpp:
```diff
@@ -27,9 +27,9 @@ hal::status application(sjsu::hardware_map& p_map)
-  auto& magnet0 = *p_map.in_pin0;
-  auto& magnet1 = *p_map.in_pin1;
-  auto& magnet2 = *p_map.in_pin2;
+  // auto& magnet0 = *p_map.in_pin0;
+  // auto& magnet1 = *p_map.in_pin1;
+  // auto& magnet2 = *p_map.in_pin2;
```
Uncomment this and all other motors.

Coreyboy1820 reviewed (changes requested):
Just uncomment the items I told you to and merge it in.

FluffyFTW committed:
drive serial uncomment

FluffyFTW committed:
:ambulance: Update DMC to MCX

- swapped arm to MC_X motors
- updated gear ratios
- updated drive serial
- fixed warnings

Coreyboy1820 reviewed (changes requested):
Change the arm code to have the correct esp connection loop.

kammce commented:
Were you able to upgrade the MC_X firmware?
